modules/33acrossBidAdapter.md

```diff
@@ -10,23 +10,145 @@ Maintainer: headerbidding@33across.com
 
 Connects to 33Across's exchange for bids.
 
-33Across bid adapter supports only Banner at present and follows MRA
+33Across bid adapter supports Banner and Video at present and follows MRA
 
 # Sample Ad Unit: For Publishers
+## Sample Banner only Ad Unit
 ```
 var adUnits = [
 {
-    code: '33across-hb-ad-123456-1', // ad slot HTML element ID   
-    sizes: [
-        [300, 250], 
-        [728, 90]
-    ],     
-    bids: [{
-        bidder: '33across',
-        params: {
-            siteId: 'cxBE0qjUir6iopaKkGJozW',     
-            productId: 'siab'     
+  code: '33across-hb-ad-123456-1', // ad slot HTML element ID  
+  mediaTypes: {
+    banner: {  
+      sizes: [
+          [300, 250], 
+          [728, 90]
+      ]
+    }   
+  } 
+  bids: [{
+    bidder: '33across',
+    params: {
+        siteId: 'sampleGUID123',     
```

Comment:
Include valid params in the README that will return data if used in a test page

Reply:
I have provided a sample GUID that is valid and will help your testing. Please let me know if you need anything more